sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
   ([#42174](https://github.com/Azure/azure-sdk-for-net/pull/42174))
 * Turned off internal spans and logs in exporter HTTP pipeline
   ([#43359](https://github.com/Azure/azure-sdk-for-net/pull/43359))
+* Logs emitted within an Activity context will now be sampled by default based on the `SamplingRatio` configured for Traces. To disable this sampling set `DisableTraceBasedSamplingForLogs` to `true`.
```

Comment:
Is there a public api for exporter?

Reply:
My bad. This should be in distro. Fixed.